suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -658,6 +658,20 @@ export const getTestnetSymbols = () => getTestnets().map(n => n.symbol);
 export const isBlockbookBasedNetwork = (symbol: NetworkSymbol) =>
     networks[symbol]?.customBackends.some(backend => backend === 'blockbook');
 
+export const isDebugOnlyAccountType = (
+    symbol?: NetworkSymbol,
+    accountType?: NonNullable<AccountType | undefined>,
+): boolean => {
+    if (!symbol) return false;
+    const network = networks?.[symbol];
+    if (!network) return false;
+
+    const accountTypeInfo =
```

Comment:
btw `isDebugOnlyAccountType` can also be on root of the object as on root of network there is default account type, but i think it is never gonna happen so ok 👍 @Lemonexe is maybe gonna refactor it anyway